decentraland/unity-renderer
# feat: implemented analytic for external link, NFTs and voicechat

## What does this PR change?

Fixes #594 and Fixes #765

This PR add analytics for tracking external links and the first time per session that the user start the voice chat recording

## How to test the changes?


1. Go to: https://play.decentraland.zone/?renderer-branch=feat/track-external-links
2. Test any external link
3. Check in the network that the analytic is been sent

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
